[2024-09-04 11:34:55,488] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 11:34:55] "OPTIONS /patient HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 11:34:55,495] WARNING add_patient() L100  Error adding patient 'Paula', Patient Paula already exists in the database - call_trace=/home/cyberforming-1790/Documentos/mvp_sprint2/api/app.py L100 
[2024-09-04 11:34:55,495] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 11:34:55] "[31m[1mPOST /patient HTTP/1.1[0m" 409 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 11:35:36,982] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 11:35:36] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 11:35:45,081] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 11:35:45] "OPTIONS /patient HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 11:35:45,096] WARNING add_patient() L100  Error adding patient 'Paula', Patient Paula already exists in the database - call_trace=/home/cyberforming-1790/Documentos/mvp_sprint2/api/app.py L100 
[2024-09-04 11:35:45,097] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 11:35:45] "[31m[1mPOST /patient HTTP/1.1[0m" 409 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:37,841] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:37] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:54,311] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:54] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:54,904] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:54] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:55,044] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:55] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:55,200] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:55] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:57,459] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:57] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:58,392] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:58] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:46:58,936] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:46:58] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:47:01,967] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:47:01] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:48:08,168] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:48:08] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:48:45,191] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:48:45] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:48:47,568] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:48:47] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:48:48,067] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:48:48] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:48:58,576] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:48:58] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:48:59,119] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:48:59] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:49:56,304] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:49:56] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:16,632] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:16] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:21,538] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:21] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:21,609] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:21] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:21,610] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:21] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:21,611] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:21] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:21,836] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:21] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:21,839] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:21] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:44,485] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:44] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:54,238] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:54] "POST /patient HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:54,248] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:54] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:50:54,252] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:50:54] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:00,922] WARNING add_patient() L100  Error adding patient 'Amanda', Patient Amanda already exists in the database - call_trace=/home/cyberforming-1790/Documentos/mvp_sprint2/api/app.py L100 
[2024-09-04 12:51:00,922] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:00] "[31m[1mPOST /patient HTTP/1.1[0m" 409 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:00,930] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:00] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:07,054] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:07] "POST /patient HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:07,062] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:07] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:07,071] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:07] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:13,505] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:13] "POST /patient HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:13,512] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:13] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:13,518] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:13] "GET /patients HTTP/1.1" 200 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
[2024-09-04 12:51:18,860] WARNING add_patient() L100  Error adding patient 'Marcos', Patient Marcos already exists in the database - call_trace=/home/cyberforming-1790/Documentos/mvp_sprint2/api/app.py L100 
[2024-09-04 12:51:18,860] INFO _log() L224  127.0.0.1 - - [04/Sep/2024 12:51:18] "[31m[1mPOST /patient HTTP/1.1[0m" 409 - - call_trace=/home/cyberforming-1790/.local/lib/python3.10/site-packages/werkzeug/_internal.py L224 
